lib/spack/spack/build_systems/lua.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+import os
+from typing import List, Text
+
+from llnl.util.filesystem import find, join_path
+
+from spack.directives import *
+from spack.package import PackageBase
+from spack.util.executable import Executable
+
+
+class LuaPackage(PackageBase):
+    """Specialized class for lua packages
+
+    Note that there is also a LuaRocksPackage for the common case where luarocks is the
```

Comment:
Where is LuaRocksPackage?

Reply:
Dang, that's a holdout from a refactor.  I initially had luapackage as a specialized MakefilePackage, but it just isn't consistent enough to be useful, so I got rid of it and promoted LuaRocksPackage to LuaPackage.